### DISCUSSION TITLE
kernel-utils PPA in Ubuntu image needed?

### DISCUSSION THREAD

**Discussion by paulmenzel** [systemd/mkosi#2616] [Q&A] [answered]
Commit https://github.com/DaanDeMeyer/mkosi/commit/b71deeba585b84f2eb362b70a45cbd22fd756a4d adds `mkosi.conf.d/20-ubuntu/mkosi.pkgmngr/etc/apt/sources.list.d/kernel-utils.sources` which is a PPA. I’d like to avoid PPAs in my Ubuntu 22.04 (jammy) image. Is there an easy way to exclude it?

**Comment by daandemeyer** [comment, marked as answer]
@paulmenzel This is from the mkosi default image. This does not end up in any image not built within the mkosi repository.

**Reply by paulmenzel** [reply]
Thank you for the instant reply. Understood. That’s what I did until now: edit the files in the mkosi archive. ;-) I created a separate directory now.
